temporal-sdk/src/main/java/io/temporal/internal/replay/ReplayWorkflowContext.java

```diff
@@ -261,5 +261,9 @@ void getVersion(
   /** Updates or inserts search attributes used to index workflows. */
   void upsertSearchAttributes(SearchAttributes searchAttributes);
 
+  /** @return workflow retry attempt */
```

Comment:
If we add this javadoc, it may be good to specify what is the stat value (0 or 1).

Reply:
added (default is 1)